Database-Features/setting-up-ad-kerberos-sso.md

```diff
@@ -124,19 +189,39 @@ Configuration is completed. Now we can test connection to the database from the
+* To verify network connectivity between the client machine and the Exasol database, first attempt to connect to the database using a standard authentication method with a username and password. For example, use the dba user created in Step 6.
   ```
   ./exaplusx64.exe -c {Full connection string to Exasol db}
   ```
   **Example**
 
   ![](images/setting-up-ad-kerberos-sso_screenshot_9.png)
 
-* Once connection is established you can be sure that client can access and proceed with testing Kerberos authentication.
-* Now add **-k** option to the command. EXAplus will ask you to type **Service name** and **Host** instead of username and password. Use **\{Exasol service name\}** and **\{Exasol host name\}** from step 3.
+* Once connection is established you can be sure that client can access the DB. Now proceed with testing Kerberos authentication.
+* Add **-k** option to the command. EXAplus will ask you to type **Service name** and **Host** instead of username and password. Use **\{Exasol service name\}** and **\{Exasol host name\}** from step 3.
 
   **Example**
 
   ![](images/setting-up-ad-kerberos-sso_screenshot_10.png)
 
+##  Known issues
+
+### Error: [28900] Cannot initialize SSPI security context
+`Error: [28900] Cannot initialize SSPI security context. The specified target is unknown or unreachable` during Step 7. This usually mean that prived parameters **Service name** and **Host** don't match the service keytab uploaded in Exasol DB.
+
+**Actions:**
+* Make sure that **Service name** and **Host** are exactly the same as returned by `setspn -L {Service account name}` command. Run it in the same command promt before runing Exaplus.
+
+
+### Error: [28900] Connection exception - authentication failed.
+`Error: [28900] Connection exception - authentication failed.` during Step 7. 
+
+**Actions:**
+* Permissions on `/var/tmp/krb5_500.rcache2`.There might be an issue with permissions for this file `/var/tmp/krb5_500.rcache2`. It is a temporary file used by the Kerberos authentication mechanism to store information about authentication tickets that have been issued. This file is generated by the Kerberos libraries when Exasol performs Kerberos authentication. Ensure that users in the `exausers` group (in particular this user: `exadefusr` ) have Read/Write permissions in the `/var/tmp/` directory on all nodes. If permission are insufficient, issue corresponding chmod command: `chmod 777 /var/tmp/`
+* Make sure that Exasol Service Account keytab is correctly imported in the DB.
+* Check that **Kerberos Realm** db parameter is correctly set in Exaoperation/EXAConf.
+* Try to regenerate keytab carefully and import it into DB once again.
+* This error could indicate several different issues. To investigate further, export the database logs following the [instructions here](https://docs.exasol.com/db/latest/administration/on-premise/admin_interface/exasupport.htm?Highlight=exasupport) and send them to Exasol Support.
```

Comment:
1.
```suggestion
* This error could indicate several different issues. To investigate further, export the database logs following the [instructions here](https://docs.exasol.com/db/latest/administration/on-premise/admin_interface/exasupport.htm) and send them to Exasol Support.
```

2. The article covers both version 7.1 with EXAoperation and version 8. Maybe we can also include a link for 7.1?